.env.example

```
@@ -475,3 +475,13 @@ LINEAR_API_KEY=
 LINEAR_TEAM_ID=
 # Game Feedback label ID in Linear (optional - find via Linear API or create a new label)
 LINEAR_GAME_FEEDBACK_LABEL_ID=
+
+# ============================================
+# NFT Minting Configuration
+# ============================================
+# Contract address for the Top 100 NFT collection (required for minting to work)
+# Must be a valid Ethereum address. Minting is disabled if not set.
+NFT_CONTRACT_ADDRESS=
+# Chain ID where the NFT contract is deployed (defaults to 1 = Ethereum mainnet)
+NFT_CHAIN_ID=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NFT_CHAIN_ID key should go before the NFT_CONTRACT_ADDRESS key

(UnorderedKey)